client/director.go

```diff
@@ -162,11 +158,15 @@ func queryDirector(ctx context.Context, verb, sourcePath, directorUrl string) (r
 		// If we get a 405 with a PROPFIND, the client will handle it
 		return
 	} else if resp.StatusCode != 307 {
-		var respErr directorResponse
+		contentType := req.Header.Get("Content-Type")
+		if contentType != "application/json" {
+			log.Debugln(string(body))
```

Comment:
Ah my bad. I meant to return a new error with `string(body)` as the error message instead. Sorry for the confusion!